okhttp/src/main/java/okhttp3/internal/Util.kt

```diff
@@ -354,7 +390,10 @@ fun Source.skipAll(duration: Int, timeUnit: TimeUnit): Boolean {
  * source is helpful, such as when doing so completes a cache body or frees a socket connection for
  * reuse.
  */
-fun Source.discard(timeout: Int, timeUnit: TimeUnit): Boolean = try {
+fun Source.discard(
```

Comment:
what’s with these changes?

Reply:
I'll revert.  I think came from the Square code formatter in Intellij?